sumpy/expansion/local.py

```diff
@@ -112,16 +114,55 @@ class VolumeTaylorLocalExpansionBase(LocalExpansionBase):
     Coefficients represent derivative values of the kernel.
     """
 
-    def coefficients_from_source(self, kernel, avec, bvec, rscale, sac=None):
-        from sumpy.tools import MiDerivativeTaker
-        ppkernel = kernel.postprocess_at_source(kernel.get_expression(avec), avec)
+    def coefficients_from_source_vec(self, kernels, avec, bvec, rscale, weights,
+            sac=None):
+        """Form an expansion with a linear combination of kernels and weights.
+
+        :arg avec: vector from source to center.
+        :arg bvec: vector from center to target. Not usually necessary,
+            except for line-Taylor expansion.
+        :arg sac: a symbolic assignment collection where temporary
+            expressions are stored.
+
+        :returns: a list of :mod:`sympy` expressions representing
+            the coefficients of the expansion.
+        """
+        from sumpy.tools import MiDerivativeTakerWrapper
```

Comment:
Add a comment comparing to `coefficients_from_source_vec` from the base class.

Reply:
Done in 6e14519c616940f80d7d562754cbb721d8fdb27e